keyboards/sofle/keymaps/devdev/config.h

```diff
@@ -0,0 +1,129 @@
+#pragma once
+
+/* The way how "handedness" is decided (which half is which),
+see https://docs.qmk.fm/#/feature_split_keyboard?id=setting-handedness
+for more options.
+*/
+
+
+//#define USE_MATRIX_I2C
+
+/* Select hand configuration */
+
+///https://thomasbaart.nl/2018/12/01/reducing-firmware-size-in-qmk/
+
+#ifndef NO_DEBUG
+#define NO_DEBUG
+#endif // !NO_DEBUG
+#if !defined(NO_PRINT) && !defined(CONSOLE_ENABLE)
+#define NO_PRINT
+#endif // !NO_PRINT
```

Comment:
This isn't actually needed
```suggestion
```

Reply:
Thanks, I was desperately short on space at some stage, and read somewhere to try that - I do seem to recall that it had no effect